rfcs/0057-FamilySeer.md

```diff
@@ -0,0 +1,212 @@
+- Feature Name: (FamilySeer: A new search method for Auto-scheduler)
+- Start Date: (2021-01-07)
+- RFC PR: [apache/tvm-rfcs#57](https://github.com/apache/tvm-rfcs/pull/57)
+- GitHub Issue: [apache/tvm#9875](https://github.com/apache/tvm/pull/9875)
```

Comment:
The link you put is the pull request but not the tracking issue. Please open a new issue to track the PR progress and put the link here.